packages/ui/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "../../tsconfig.base.json"
+}
```

Comment:
Could you please remove the `lib.json` file and refer `spec.json` here?
```suggestion
{
  "extends": "../../tsconfig.base.json",
  "references": [
    {
      "path": "./tsconfig.spec.json"
    }
  ]
}
```

Reply:
@bc-peng Should we do the same for each package that have `lib.json` file?

Reply:
@bc-peng Should we make the same changes for each package that we have already?

Reply:
done for UI package

Reply:
@serhii-tkachenko We adpated this in the middle of TFA, and at the end of the project, we will clean up all packages and follow this pattern.